content/introduction/installation/docker.md

```diff
@@ -147,8 +122,165 @@ db_1  | TRACE: returning result row [INT64(0)]
+- `server` has a single `.yaml` file and for server configuration
+
+```bash
+shared > ls
+databases	dolt		server
+shared > docker run -p 3307:3306 -v $PWD/server:/etc/dolt/servercfg.d -v $PWD/dolt:/etc/dolt/doltcfg.d -v $PWD/databases:/var/lib/dolt dolthub/dolt-sql-server:latest 
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: Entrypoint script for Dolt Server 0.50.10 starting.
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: Checking for config provided in /etc/dolt/doltcfg.d
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: /etc/dolt/doltcfg.d/config.json file is found
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: Checking for config provided in /etc/dolt/servercfg.d
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: /etc/dolt/servercfg.d/config.yaml file is found
+
+
+2022-10-27 18:07:51+00:00 [Warn] [Entrypoint]: /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
+2022-10-27 18:07:51+00:00 [Note] [Entrypoint]: Dolt Server 0.50.10 is started.
+Starting server with Config HP="0.0.0.0:3306"|T="28800000"|R="false"|L="debug"
+
+```
+
+We can see both the config file was used successfully.
+
+We can verify that we have the data we create through the server in our local directory we mounted.
+
+```bash
+> mysql --host 0.0.0.0 -P 3307 -uroot
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 1
+Server version: 5.7.9-Vitess 
+
+Copyright (c) 2000, 2022, Oracle and/or its affiliates.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+mysql> create database doltdb;
+Query OK, 1 row affected (0.08 sec)
+
+mysql> use doltdb;
+Database changed
+mysql> create table mytable(pk int primary key, col1 varchar(20));
+Query OK, 0 rows affected (0.02 sec)
+
+mysql> insert into mytable values (1, 'first row'),(2, 'second row');
+Query OK, 2 row affected (0.02 sec)
+
+mysql> exit
+```
+
+We can check for directory `doltdb` created in our local `/shared/databases` directory. 
+
+```bash
+shared > cd databases && ls
+doltdb
+databases > cd doltdb && dolt sql -q "SELECT * FROM mytable"
++----+------------+
+| pk | col1       |
++----+------------+
+| 1  | first row  |
+| 2  | second row |
++----+------------+
+
+```
+
+You can verify it has the data we created by using Dolt CLI Docker image if you do not have Dolt installed locally.
+
+```bash
+shared > cd databases && ls
+doltdb
+databases > docker run -v $PWD/doltdb:/doltdb dolthub/dolt:latest sql -q "USE doltdb; SELECT * FROM mytable;"
+Database changed
++----+------------+
+| pk | col1       |
++----+------------+
+| 1  | first row  |
+| 2  | second row |
++----+------------+
+
+```
+
 ## Sample Dockerfiles <a name="samples"></a>
 
+Here are a simple Dockerfiles that installs the latest version of Dolt to start with if you want customized Dockerfile for your use case.
+
```

Comment:
I'm probably just rehashing this... but... now that we have the official images, I'm not sure this info is as valuable anymore. We can point people to the official images and they can use them as an example if they want to create a custom image based off a different linux distribution or something.